param/parameterized.py

```diff
@@ -1129,19 +1129,25 @@ def watch(self_,parameter_name,parameter_attribute=None,fn=None):
                 subscribers[parameter_name] = {}
             if parameter_attribute not in subscribers[parameter_name]:
                 subscribers[parameter_name][parameter_attribute] = []
-            subscribers[parameter_name][parameter_attribute].append(fn)
+            getattr(subscribers[parameter_name][parameter_attribute],action)(fn)
         else:
             subscribers = self_.cls.params(parameter_name).subscribers
             if parameter_attribute not in subscribers:
                 subscribers[parameter_attribute] = []
-            subscribers[parameter_attribute].append(fn)
+            getattr(subscribers[parameter_attribute],action)(fn)
 
+    def watch(self_,fn,parameter_name,parameter_attribute=None):
+        self_._watch('append',fn,parameter_name,parameter_attribute)
 
+    def unwatch(self_,fn,parameter_name,parameter_attribute=None):
+        self_._watch('remove',fn,parameter_name,parameter_attribute)
+
+    # TODO: now unused?
```

Comment:
@philippjfr shall I delete subscribe() from this PR? Or do you use it?

Reply:
I haven't been using it, deleting it sounds good to me.